test/test_bulk.py

```diff
@@ -376,6 +379,27 @@ def test_client_generated_upsert_id(self):
                           {'index': 2, '_id': 2}]},
             result.bulk_api_result)
 
+    def test_upsert_uuid_standard(self):
+        options = CodecOptions(uuid_representation=UuidRepresentation.STANDARD)
```

Comment:
Nice. Could you add a similar test to ensure that UuidRepresentation.UNSPECIFIED results in Binary instances?

Reply:
As far as I can tell we can't do that because it raises a [`ValueError`](https://github.com/mongodb/mongo-python-driver/blob/308b4f4e08c9fe91210ee498adbbafe0c7ebebe4/bson/binary.py#L253).

Reply:
Yes the test will need to convert the uuids to Binary manually before inserting them:
```
uuids = [Binary.from_uuid(uuid.uuid4()) for _ in range(3)]
```

Reply:
Done